index.rst

```diff
@@ -44,18 +44,589 @@
+- PhoSim :cite:`2015ApJS..218...14P` (This is used for closed-loop simulations)
+
+In order for the AOS to work as one system, a clean understanding of these CS and how they fit together is required. This is the major objective of this technical note. We will also discuss how to transform coordinates and operations between various CS, and more importantly, how to transform everything into a common CS - the Optical CS.
+
+.. note::
+
+   Document-18499 :cite:`Document-18499` defines a set of CS and the transformations between them. However, it doesn't go into all the levels of details that are sufficient for understanding the AOS operations.
+
+.. _section-ocs:
+
+#############################
+The Optical Coordinate System
+#############################
+
+Our AOS system-wide CS is the Optical Coordinate System (OCS). The defition of the OCS is the same as defined in LTS-136 :cite:`LTS-136`, Section 2.2. In summary -
+
+- The origin of the OCS is at the theoretical vertex of M1.
+- +z axis points along the optical axis, toward the sky
+- +x axis goes in parallel with the elevation axis. When look from the sky, +x points to the right.
+- +y axis can then be determined using the right hand rule. At horizon pointing, +y goes up.
+- The azimuth angle is zero when the telescope points toward northern horizon. Looking from the sky, it increase clockwise.
+- The zenith angle is zero when the telescope points at zenith. It increase to 90 degrees for horizon pointing.
+
+.. figure:: /_static/ocs.png
+   :name: fig-ocs
+   :target: ../_images/ocs.png
+   :alt: The Optical Coordinate System (OCS)
+
+Optical Coordinate System (OCS). Left: view from the sky; Right: side view from the -X axis.
+
+The idea is that the OCS is fixed to M1, which defines the reference for the optical system.
+Relative to the ground, the OCS rotates with the azimuth angle and the zenith angle.
+An easy way to identify the +x/+y/+z while looking at a drawing or the actual hardware is that, the walkway on the TMA, above M1M3, is in the +y direction. Otherwise when we go toward horizon it is going to bump into the ground.
+
+.. note::
+   The Telescope and Site top-level CS document is LTS-136 :cite:`LTS-136`, and the Camera top-level CS document is LCA-280 :cite:`LCA-280`. Unlike those two documents, we do not define the Observatory Mount CS or the Azimuth CS, because these CS, as defined by LTS-136 :cite:`LTS-136` and LCA-280 :cite:`LCA-280`, actually contradict each other, in terms of which axis points north, which points west etc. We also avoid the use of terms such as ``elevation angle`` and ``altitude``. Instead, we always use ``zenith angle``, as defined above.
+
+
+################
+Zemax and PhoSim
+################
+
+We discuss the Zemax and PhoSim CS first, because these are relatively easier to define -
+they are self-consistent within their own framework.
+When we say Zemax CS, we refer to the global CS as used by
```

Comment:
Which "global CS" you are talking about here:
https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=SYSENG&title=As-built+optical+model

Reply:
The confluence page links to zmx file. You will need to open the zmx file to see the CS it uses.

Reply:
Not everyone contains me has the ZEMAX license. Is it possible to explain it in the document directly? Thanks!

Reply:
The document already defines ZCS.